tools/WhitelistItems/SiriusWalletsService.cs

```diff
@@ -0,0 +1,341 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Common;
+using Common.Log;
+using Google.Protobuf.WellKnownTypes;
+using Lykke.Common.Log;
+using Polly;
+using Polly.Contrib.WaitAndRetry;
+using Swisschain.Sirius.Api.ApiClient;
+using Swisschain.Sirius.Api.ApiContract.Account;
+using Swisschain.Sirius.Api.ApiContract.User;
+using Swisschain.Sirius.Api.ApiContract.WhitelistItems;
+
+namespace WhitelistItems
+{
+    public class SiriusWalletsService
```

Comment:
can we use this class from the HFT API itself?

Reply:
it has too many unused in tool dependencies